<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeighborhoodDialog</class>
 <widget class="QDialog" name="NeighborhoodDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate neighborhood</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout1">
       <item row="0" column="0">
        <widget class="QLabel" name="labelOrigin">
         <property name="text">
          <string>Neighborhood origin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelOriginFormula">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonChemOperSelector">
         <property name="text">
          <string>Choose morphing operators</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelFingerprintSelector">
         <property name="text">
          <string>Fingerprint method</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboFingerprintSelector"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelSimCoeffSelector">
         <property name="text">
          <string>Similarity coefficient method</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboSimCoeffSelector"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelDimRedSelector">
         <property name="text">
          <string>Visualization method</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboDimRedSelector"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelAttemptCount">
         <property name="text">
          <string>Neighborhood size</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBoxAttemptCount">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelMaxDepth">
         <property name="text">
          <string>Maximal morphing depth</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="spinBoxMaxDepth">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelMaxDistance">
         <property name="text">
          <string>Maximal distance from origin</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxDistance">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.200000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelContext">
         <property name="text">
          <string>Visualization context</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetContext">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
